openshift/patroni.dc.yaml

```diff
@@ -368,12 +368,12 @@ parameters:
     description: Starting amount of memory the container can use.
     displayName: Memory Request
     required: true
-    value: 256Mi
+    value: 384Mi
   - name: MEMORY_LIMIT
     description: Maximum amount of memory the container can use.
     displayName: Memory Limit
     required: true
-    value: 512Mi
+    value: 768Mi
```

Comment:
Note that this will be applied to dev, test, and prod. This isn't necessarily a bad thing, _unless_ dev or test don't use the `request` amount. In that case we can do something like a per-environment parameter file for these settings.